tgui/packages/tgui/interfaces/MessageMonitor.tsx

```diff
@@ -0,0 +1,370 @@
+import { BooleanLike } from 'common/react';
+import { useBackend, useLocalState } from '../backend';
+import { Section, Stack, Input, Button, Table, NoticeBox, Box } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  screen: number;
```

Comment:
Change this to a numeric enum.

https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums